### DISCUSSION TITLE
Match admin column heading "Title" with its Label

### DISCUSSION THREAD

**Discussion by dr-codswallop** [craftcms/cms#12500] [Ideas] [answered]
Would be great to see the heading for **Title** admin column equate to its **Label** to cater for cases where we've relabelled the default title field. 

Example case for entries used as "locations"

**Field layout**
<img width="296" alt="image" src="https://user-images.githubusercontent.com/49959728/210612364-1c1956dd-2f9b-482a-912f-a526434143b2.png">

**Entry form**
<img width="430" alt="image" src="https://user-images.githubusercontent.com/49959728/210612759-a481faa3-4468-4c83-bf60-e3b18c651783.png">

**Entry list**
Instead of the heading word "Title" I think it's more intuitive show the field's label "Location" to accurately reflect its display  naming elsewhere.

<img width="380" alt="image" src="https://user-images.githubusercontent.com/49959728/210612585-d1240c90-622d-403b-97eb-ddea9337aa94.png">

**Comment by brandonkelly** [comment, marked as answer]
This was previously requested by #12032. Explained there why it’s not the case by default, and how you can change it with some PHP code.
